lib/Twig/Template.php

```diff
@@ -126,6 +126,34 @@ public function displayParentBlock($name, array $context, array $blocks = array(
     }
 
     /**
+     * Checks if a block exists.
+     *
+     * This method is for internal use only and should never be called
+     * directly.
```

Comment:
the long description should also describe the difference between this method and ``hasBlock``